README.md

```diff
@@ -13,41 +13,28 @@ _Create workflows that enable you to use Continuous Integration (CI) for your pr
 
 </header>
 
-<!--
-  <<< Author notes: Step 4 >>>
-  Start this step by acknowledging the previous step.
-  Define terms and link to docs.github.com.
--->
+## Step 5: Merge your pull request
 
-## Step 4: Add branch protections
+_Almost there! :heart:_
 
-_Great job uploading test reports! :partying_face:_
+You can now [merge](https://docs.github.com/get-started/quickstart/github-glossary#merge) your pull request!
 
-Take a look at the merge box, you'll notice you can merge this even though the review process hasn't been met.
+### :keyboard: Activity: Merge your pull request
 
-Protected branches ensure that collaborators on your repository cannot make irrevocable changes to branches. Enabling protected branches also allows you to enable other optional checks and requirements, like required status checks and required reviews.
+1. Go to the **Pull requests** tab.
+1. Resolve any merge conflicts. <!-- TODO: figure out a better way to deal with the merge conflicts. -->
+1. Click **Merge pull request**.
+1. _Once you turn on branch protection, Actions can no longer push directly to the `main` branch. Make sure that you're on the `ci` branch in the page you're following instructions from._ Wait about 20 seconds and then refresh the page. [GitHub Actions](https://docs.github.com/actions) will automatically update to the next step.
```

Comment:
Seems that very often things need to be exactly written for the test to pass. This is not realistic in real-life, even though meant as a demonstration of exactly that. No freedom available to do anything even slightly differently from the instructions. Later the instructions change to something very abstract and following the ruleset is impossible and impossible not to follow at the same time.